src/System.Private.Xml.Linq/src/System/Xml/Linq/XElement.cs

```diff
@@ -472,7 +472,7 @@ public IEnumerable<XElement> DescendantsAndSelf(XName name)
         public XNamespace GetDefaultNamespace()
         {
             string namespaceName = GetNamespaceOfPrefixInScope("xmlns", null);
-            return namespaceName != null ? XNamespace.Get(namespaceName) : XNamespace.None;
+            return namespaceName != null ? XNamespace.Get(namespaceName) : XNamespace.Get(string.Empty);
```

Comment:
why is `XNamespace.None` not ok to be returned?

Reply:
yeah, actually this was unnecessary.